API/Controllers/UserController.cs

```diff
@@ -25,5 +26,13 @@ public async Task<ActionResult> GetUserInfo()
             var userInfo = await _userService.GetUserProfileInfoAsync(userId);
             return Ok(userInfo);
         }
+
+        [HttpPost("edit-info")]
+        public async Task<ActionResult> UserEditProfileInfo(UserEditProfileInfoDTO userEditProfileInfo)
+        {
+            var userId = _userService.GetCurrentUserNameIdentifier(User);
+            await _userService.UserEditProfileInfoAsync(userEditProfileInfo, userId);
+            return Ok();
```

Comment:
Should we return user info after editing to update it on front?

Reply:
For what?

Reply:
To update it on front

Reply:
We don't need it